test/resources/logback-test.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I was thinking that when running tests logback would search for logback-test.xml and if not found fallback to logback.xml but never tested it though